common/src/main/java/com/wynntils/features/WynncraftButtonFeature.java

```diff
@@ -85,13 +68,13 @@ public void renderButton(@NotNull PoseStack matrices, int mouseX, int mouseY, fl
         }
 
         public static void onPress(Button button) {
-            if (button instanceof WynncraftButton wynncraftButton) { // TODO is check necessary
-                ConnectScreen.startConnecting(
-                        wynncraftButton.backScreen,
-                        Minecraft.getInstance(),
-                        ServerAddress.parseString(wynncraftButton.serverData.ip),
-                        wynncraftButton.serverData);
-            }
+            if (!(button instanceof WynncraftButton wynncraftButton)) return;
```

Comment:
The TODO has been removed, so can you confirm its necessary?

Reply:
We need to cast the `Button` to `WynncraftButton` to be able to access its methods. So we might as well do the casting conditionally. In practice the button will always be a `WynncraftButton` though.